man/bootctl.xml

```diff
@@ -237,6 +237,19 @@
         cannot be written. Currently only applies to random seed operations.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--make-machine-id-directory=yes|no|auto</option></term>
+        <listitem><para>Control creation and deletion of <literal>$ESP/$MACHINE_ID</literal> during
+        <option>install</option> and <option>remove</option>, respectively.
```

Comment:
This does not really say what the effect of the option is. What happens when "no" and the directory does not exist? This option has a very important effect on subsequent operation of `kernel-install`, and this effect needs to be mentioned here.

Reply:
Or maybe just say "See kernel-install(8) for a discussion of the effects of the presence of this directory."

Reply:
I mean, to me "Control creation and deletion of `$ESP/$MACHINE_ID` during `install` and `remove`, respectively." reads pretty unambiguously as "install => yes => create `$ESP/$MACHINE_ID`", "remove => yes => delete `$ESP/$MACHINE_ID`".

The entirety of the discussion is in the paragraph describing `$BOOT` detexion, which is moot here since we always create `$BOOT/loader/entries`.

Reply:
OK.